public/docs/ts/latest/guide/architecture.jade

```diff
@@ -392,7 +391,7 @@ block dart-bool
 
 +makeExcerpt('app/hero-detail.component.html', 'ngModel')
 :marked
-  Angular includes a small number of other directives that either alter the layout structure
+  Angular contains a small number of other directives that either alter the layout structure
```

Comment:
This is good, but on second reading, I thought it could be shorter:

Angular has a few more directives...

But we could endlessly fiddle with the text. What you have is good enough.